src/common/helpers.js

```diff
@@ -55,3 +55,7 @@ export async function asyncFetch(url, params) {
   if (!response.ok) throw new Error(response.status);
   return await response.json();
 }
+
+export function getAmazonDownloadLinkUrl(s3_bucket, s3_key) {
+  return `https://s3.amazonaws.com/${s3_bucket}/${s3_key}`;
```

Comment:
I'm guessing this is just a temporary placeholder for now?

Reply:
Depends on how we're going to generate the download link?? I didn't see any field with that information. Instead what the backend is sending is: `s3_bucket` and `s3_key`. I'm guessing the final download url will look like that?